exercises/04-react-redux/src/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import { Root } from 'modules/root/components/root'
```

Comment:
the import order is a thing of minuscule priority and we don't strictly follow it, but it's good to have on mind that first you import third party libraries and only then you import local stuff

Reply:
I think we should add an ESLint rule if we want to follow it.

Reply:
Does such rule exist? I see only the ['sort-imports'](https://eslint.org/docs/rules/sort-imports) rule, which 

> verifies that all imports are first sorted by the used member syntax and then alphabetically by the first member or alias name

Reply:
Yes, there are these plugins:

- https://github.com/Tibfib/eslint-plugin-import-helpers
- https://github.com/lydell/eslint-plugin-simple-import-sort

Not sure what is exactly different.

Reply:
I think the second one can also sort named imports between `{` and `}`. Not sure if we need it. :)

Reply:
The change will affect all exercises, so I propose to address it in a separate [issue](https://github.com/salsita/react-training/issues/56)

Reply:
Fine with me. Let's just try to follow it going forward ;)

Reply:
I've reorganized the imports manually for now. See 0ee4a3c